Challenge 2/Challenge 2/GameManager.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  GameManager.swift
+//  Challenge 2
+//
+//  Created by Bartosz Lipiński on 17/04/2023.
+//
+
+import Combine
+
+class GameManager: ObservableObject {
+    @Published var selectedMoveByApp = PossibleMove.allCases.randomElement() ?? .rock
+    @Published var winningMode = Bool.random()
+    @Published var score = 0
+    @Published var questionCount = 0
+    @Published var postMoveAlertTitle = ""
+    @Published var postMoveAlertMessage = ""
```

Comment:
doesn't have to be @Published as showPostMoveAlert is already refreshing view's body. Make sure to set it before showPostMoveAlert